src/conn/conn_tiered.c

```diff
@@ -180,6 +182,7 @@ __wt_tier_do_flush(
     WT_DECL_RET;
     WT_FILE_SYSTEM *bucket_fs;
     WT_STORAGE_SOURCE *storage_source;
+    uint32_t millis, tries;
```

Comment:
Minor: WT uses `msecs` or `msec` generally. And `retry`  or `retries` instead of tries.

Reply:
Fixed.